### PR TITLE
[Acceptance Test] Add e2e tests for scenarios adding new job application

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
This PR adds e2e tests for testing different scenarios involved in adding new job application form.

### CODE REVIEW

File: frontend/tests/acceptance/helpers/api/job_application.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  deleteJobApplication: async (token) => {
+    axios.defaults.headers.common['Authorization'] = `Bearer ${token}`;
+    try {
+      jobs = await getJobApplications(token);
+      if (jobs.length != 0) {
+        jobs.forEach(async (job) => {
+          const priority = getPriorityID(job.priority);
+          try {
+            await axios.delete(`${serverUrl}/applications/${job.id}`, {
+              params: { title: job.title, status: job.status, priority: priority, company_name: job.company },
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply frontend/tests/acceptance/helpers/api/job_application.js:15]: axios recommends params. so i used them. will refine

[the remark this answers is not in the document]
Comment (koushikmohan1996) [reply frontend/tests/acceptance/helpers/api/job_application.js]: What I am referring is delete application will work without sending any additional params. `axios.delete(`${serverUrl}/applications/${job.id})` This should work without any issues

[the remark this answers is not in the document]
Comment (saw-jan) [reply frontend/tests/acceptance/helpers/api/job_application.js]: ohh. i tried once but couldn't delete so tried with ids. will try again

[the remark this answers is not in the document]
Comment (saw-jan) [reply frontend/tests/acceptance/helpers/api/job_application.js]: this fails
`await axios.delete(`${serverUrl}/applications`);` 

this also fails
`await axios.delete(`${serverUrl}/applications`, {
              params: { title: job.title, status: job.status, priority: priority, company_name: job.company },
            });`

[the remark this answers is not in the document]
Comment (koushikmohan1996) [reply frontend/tests/acceptance/helpers/api/job_application.js:15]: You missed job.id. Its important

[the remark this answers is not in the document]
Comment (koushikmohan1996) [reply frontend/tests/acceptance/helpers/api/job_application.js:15]: Anyway, not a big deal. Merging this PR.

**Pull request merged**